app/indexers/digital_instantiation_indexer.rb

```diff
@@ -18,6 +18,7 @@ def generate_solr_document
         solr_doc['instantiation_admin_data_tesim'] = object.instantiation_admin_data.gid if !object.instantiation_admin_data.gid.blank?
         solr_doc['aapb_preservation_lto_ssim'] = solr_doc['aapb_preservation_lto_tesim'] = object.instantiation_admin_data.aapb_preservation_lto if !object.instantiation_admin_data.aapb_preservation_lto.blank?
         solr_doc['aapb_preservation_disk_ssim'] = solr_doc['aapb_preservation_disk_tesim'] = object.instantiation_admin_data.aapb_preservation_disk if !object.instantiation_admin_data.aapb_preservation_disk.blank?
+        solr_doc['md5_tesim'] = object.instantiation_admin_data.md5 if !object.instantiation_admin_data.md5.blank?
```

Comment:
It makes more sense to use `_ssim` rather than `_tesim` here.

Reply:
👍 - i'll make the change and make sure tests run.

Reply:
should be good to go.